assets/quick-order-list.js

```diff
@@ -178,13 +176,26 @@ if (!customElements.get('quick-order-list')) {
           event.target.setCustomValidity('');
           event.target.reportValidity();
           if (cartQuantity > 0) {
-            this.updateQuantity(index, inputValue, name, this.actions.update);
+            this.startQueue(index, inputValue);
           } else {
-            this.updateQuantity(index, quantity, name, this.actions.add);
+            this.startQueue(index, quantity);
           }
         }
       }
 
+      // startQueue(id, quantity) {
+      //   this.queue.push({id, quantity})
+      //   const int = setInterval(() => {
+      //     if (this.queue.length > 0) {
+      //       if (!this.requestStarted)  {
+      //         this.sendRequest(this.queue)
+      //       }
+      //     } else {
+      //       clearInterval(int)
+      //     }
+      //   }, 250)
+      // }
```

Comment:
🔥?

Reply:
Yes :) just wanted to get some thoughts on the extendable component. But deleting for now